src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTabHelper.js

```diff
@@ -0,0 +1,113 @@
+({
+    /**
+     * @description: instantiates component. Only called when component is first loaded.
+     */
+    doInit: function(component) {
+        var action = component.get('c.isOrgNamespaced');
+        action.setCallback(this, function (response) {
+            var state = response.getState();
+            if (state === 'SUCCESS') {
+                var isOrgNamespaced = JSON.parse(response.getReturnValue());
+                if (isOrgNamespaced) {
+                    var listViewNamespaced = {
+                        'objectApiName': 'npsp__DataImportBatch__c',
```

Comment:
Is this fixed in @randi274's namespace branch?

Reply:
Yes!